modules/core/src/test/java/org/apache/ignite/testframework/wal/record/RecordUtils.java

```diff
@@ -564,4 +564,14 @@ public static MvccTxRecord buildMvccTxRecord() {
     public static ConsistentCutRecord buildConsistentCutRecord() {
         return new ConsistentCutRecord();
     }
+
+    /**
+     * Return {@code true} if append to write-ahead log.
+     *
+     * @param walRecord Instance of {@link WALRecord}.
+     * @return {@code True} if append to write-ahead log.
+     */
+    public static boolean isLogEnabled(WALRecord walRecord) {
```

Comment:
It's better to change this method's name and javadoc, now it's purpose is not obvious (e.g. "isIncludedInLog").